### PR TITLE
fix(typedoc): removed legacy typedoc config #551

### PR DESCRIPTION
**Pull request by mancioshell**

### CODE REVIEW

File: src/AuthContext.tsx [hunk 1]
```diff
@@ -27,6 +28,27 @@ export const hasCodeInUrl = (location: Location): boolean => {
   );
 };
 
+export const errorInUrl = (location: Location): Error | null => {
```
Comment (cobraz) [inline src/AuthContext.tsx:31]: This should probably not be a part of this pull request (as it is part of #552)

**Review by cobraz — CHANGES_REQUESTED** [review]
We probably don't want more the typedoc changes, but other than that it all looks good to me!

File: src/AuthContext.tsx (hunk 1 above)
Comment (pamapa) [reply src/AuthContext.tsx:31]: Should i help rebasing this, such that it can land anytime soon?

Comment (pamapa) [reply src/AuthContext.tsx:31]: prepared in #582

**Pull request closed without merging**